.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -14,13 +14,20 @@ body:
       options:
         - label: The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.
           required: true
+        - label: The bug is not caused by stale cache. It still happens after clearing extension cache.
+          required: true
```

Comment:
Maybe it should mention where to find the "Clear cache" button?

Reply:
I'm not good at wording. Do you have any precise example?

Reply:
```suggestion
        - label: The bug is not caused by a stale cache. It still happens after clearing the extension cache.
          description: You can find the "Clear cache" button at the bottom of the options page.
          required: true
```

Reply:
> Not sure if `description` works with checkboxes…

It doesn't work, so I put your words in the label directly.